pkg/rancher-desktop/main/tray.ts

```diff
@@ -126,7 +130,7 @@ export class Tray {
     const paths = await kubeconfig.getKubeConfigPaths();
     const options: fs.WatchOptions = {
       persistent: false,
-      recursive:  true,
+      recursive:  !this.isLinux(),
```

Comment:
Please add a comment to say why not Linux ("because it's not implemented" is fine).

Reply:
Comment added to provide more context.